projects/core/src/auth/http-interceptors/client-token.interceptor.spec.ts

```diff
@@ -22,22 +23,22 @@ const testToken = {
   scope: ''
 } as ClientToken;
 
-class MockAuthService {
+class MockAuthService extends AuthService {
   getClientToken(): Observable<ClientToken> {
     return of();
   }
 }
 
-const MockAuthModuleConfig: AuthConfig = {
-  server: {
+class MockAuthModuleConfig extends AuthConfig {
```

Comment:
The same applies here as I said for extending services. In this case, using a `const` value with a type is sufficient.